src/Files.App/Converters/ObjectToBoolConverter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Data;
+using System;
+
+namespace Files.App.Converters
+{
+	public class ObjectToBoolConverter : IValueConverter
```

Comment:
I'm pretty sure we already have a converter for this, either as part of the Toolkit or something custom.

Reply:
Oh that's right. I'm going to remove them.